go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -4634,3 +4634,148 @@ var DoltVerifyConstraintsTestScripts = []queries.ScriptTest{
 		},
 	},
 }
+
+var DoltTagTestScripts = []queries.ScriptTest{
```

Comment:
These go enginetests are awesome! Thanks for adding them. 

It would be nice to have some coverage over other columns in the `dolt_tags` table. tagger and email should be set to test values that don't change. date and tag_hash will change with each run of course, but you could still assert that they are not null.